deployment/image-index.yml

```diff
@@ -0,0 +1,4 @@
+images:
+  - name: woolpret-geonode
+    dockerfile: deployment/docker/Dockerfile
```

Comment:
I suppose we would need to add this `dockerfile` or it's not necessary ?

Reply:
We need the dockerfile that will be used to build the image. This config is the same as IHE Delft.

Reply:
Will take a look.